src/4.3_to_5.0/migrate_43_to_50.ts

```diff
@@ -211,33 +208,23 @@ const accumulateStructure = ({
 export async function migrate_43_to_50(
   legacyUIFolder: ContentRecord,
   {
+    migratedUIFolder,
```

Comment:
It seems wrong to have both `legacyUIFolder` and `migratedUIFolder`.
I thought we had agreed on passing `contentServer` down the line, and that each function would mutate it.
If so then here we shouldn't have en extra `migratedUIFolder` argument, and `legacyUIFolder` should be renamed to `uiFolder`